Last-Update: 2019-04-22
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: attempted fix for FTBFS on hurd-i386.
```
 xpra/monotonic_ctime.c:20:10: fatal error: mach/clock.h: No such file or directory
  #include <mach/clock.h>
           ^~~~~~~~~~~~~~
```

--- a/xpra/monotonic_ctime.c
+++ b/xpra/monotonic_ctime.c
@@ -15,9 +15,9 @@
 #include <time.h>
 #include <sys/time.h>
 #include <stdio.h>
 
-#ifdef __MACH__
+#ifdef __APPLE__
 #include <mach/clock.h>
 #include <mach/mach.h>
 #endif
 #endif
@@ -27,9 +27,9 @@
 #ifdef _WIN32
 	ULONGLONG ticks = GetTickCount64();
 	return ((double) ticks) / 1000;
 #else
-#ifdef __MACH__
+#ifdef __APPLE__
 	clock_serv_t cclock;
 	mach_timespec_t mts;
 	host_get_clock_service(mach_host_self(), SYSTEM_CLOCK, &cclock);
 	clock_get_time(cclock, &mts);
